src/style/Navbar.css

```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+.navbar {
+  width: 87.5rem;
+  height: 5.938rem;
+
+  /* margin: 0 0 2.375rem; */
```

Comment:
- [ ] It would be better if we remove the commented out code, it makes our codebase messy